src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -130,52 +160,33 @@ const NewFormRuleDialog: React.FC<Props> = ({
     },
   });
 
-  // This would be better handled with real form validation
-  const [validationError, setValidationError] = useState<string>();
-  const handleSubmit = () => {
-    let error: string | undefined = undefined;
-    if (formRole === FormRole.Service) {
-      if (rule.serviceCategoryId && rule.serviceTypeId) {
-        error =
-          'You cannot choose both Service Category and Service Type. Please only choose one';
-      } else if (!(rule.serviceCategoryId || rule.serviceTypeId)) {
-        error = 'One of either Service Category or Service Type are required';
```

Comment:
It looks like we lost this validation error, which we should keep. I'm able to create a service rule without specifying any service type/category

Reply:
Thanks, good point, I added this back. The 'cannot choose both' error is no longer needed, but we still need 'one of them is required'

Is there a better way to do this? I was curious about @ttoomey's comment that this would be better handled using real form validation. Should we be trying to standardize around using RHF for forms like this?

Reply:
Yes probably, but we are only using RHF inside the form builder right now and we haven't really hardened off our conventions around validation there either. I think that woudl be ideal, though, that we use RHF for most form inputs including the integrated client side validation